### ISSUE TITLE
`iuc_pr` sub-command.

### ISSUE THREAD

**Issue by bgruening** [galaxyproject/planemo#477]
I would like to have a easy way to create a PR to the IUC repository to make it easier for users to contribute their work.

**Comment by jmchilton** [comment]
I like this idea - @nturaga wanted to do something like this for bioconda I think.  Can you spec out what you want it to do - I might be willing to take a crack at implementing it. Is it going to just open a pull request from an existing branch, is it going to do IUC checks, etc...?

I think I was imagining an ``tool_branch --[iuc|galaxyp|devteam|bgruening]`` command to start the process. Maybe we could do ``planemo branch_init`` and  ``planemo branch_pr`` for consistency.

**Comment by bgruening** [comment]
I would like to have the following workflow:
* init
* lint
* (conda)
* test
* push to IUC/GalaxyP...

The command should ask for github credentials and fork/clone the appropriate repository, create a branch with the repository name from shed.yml and push it. Ideally return the link to the github PR.

I would keep this simple and not split it up in multiple commands. If someone knows the ins and outs of git I suppose he will use it directly. I feel planemo already has to many commands with overlapping functionality and gets more and more confusing for users.

**Comment by nturaga** [comment]
@bgruening @jmchilton yes, this was part of the plan, to pr to bioconda after creating the recipe.
